bpf/lib/nat.h

```diff
@@ -742,7 +743,9 @@ snat_v4_prepare_state(struct __ctx_buff *ctx, struct ipv4_nat_target *target)
 		return NAT_NEEDED;
 	}
 # endif
-#else
+#endif /* TUNNEL_MODE && IS_BPF_OVERLAY */
+
+#if defined(IS_BPF_HOST)
```

Comment:
Nit:

```suggestion
#ifdef IS_BPF_HOST
```

(and same below)